src/platform_impl/linux/x11/window.rs

```diff
@@ -371,15 +371,15 @@ impl UnownedWindow {
                     } else {
                         max_inner_size = Some(dimensions.into());
                         min_inner_size = Some(dimensions.into());
-
-                        let mut shared_state = window.shared_state.get_mut();
-                        shared_state.min_inner_size = window_attrs.min_inner_size;
-                        shared_state.max_inner_size = window_attrs.max_inner_size;
-                        shared_state.resize_increments = pl_attribs.resize_increments;
-                        shared_state.base_size = pl_attribs.base_size;
                     }
                 }
 
+                let mut shared_state = window.shared_state.get_mut();
```

Comment:
I'm not quite sure how commit message refers to dpi change in the first place. From what I can see they were never called in the first place for a resizable window, which is a bug.

So you must state that in `CHANGELOG.md` and commit message, since what you're telling fixing is not what you've actually fixed.

Something like `On X11, fix min, max, and resize increments not being applied for resizable window.`

Reply:
Given how bad our X11 backend is I think I got why it works sometimes now. It basically sets the attributes but not keep them since the window is resizable.

`On X11, fix min, max, and resize increments not persist for resizable window.`

That is a bit more clear, I think.

Reply:
Done